.cirrus.yml

```diff
@@ -0,0 +1,18 @@
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+task:
+  name: FreeBSD
+  env:
+    matrix:
+      - JULIA_VERSION: 1.0
+      - JULIA_VERSION: 1.1
+      - JULIA_VERSION: 1.2
+      - JULIA_VERSION: 1.3
+      - JULIA_VERSION: nightly
+  allow_failures: JULIA_VERSION == 'nightly'
```

Comment:
I think this needs to be `$JULIA_VERSION` ([ref](https://github.com/cirruslabs/cirrus-ci-docs/commit/7d389ea688111ac2cc88aea12a15e7ae1756794e))

Reply:
Let's try. There was no complaint about the syntax.